apps/parachain-tracker/src/app/home/home.component.scss

```
@@ -0,0 +1,9 @@
+section {
+    /* Rectangle 2 */
+    width: 100%;
+    height: 112px;
+
+    /* Subtle */
+
+    background: #D3B9C3;
```

Can use sass variable here
```suggestion
    background: $subtle-color;
```